ui/react-components/utils/AppointmentSMS.js

```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash'
```

Comment:
Shouldn't this file be in Appointment Frontend module repository?

Reply:
Please validate with @binduak

Reply:
We confirmed with Bindu and this is the repository for appointment module frontend